chapters/tutorials/index.rst

```diff
@@ -116,18 +123,16 @@ So now we have a Flow with two elements. Those elements are two `Executors`. We
         def bar(self, **kwargs):
             print(f'bar is doing cool stuff: {kwargs}')
 
-We will have more complex Executors later, for now, the only important part for you to understand is that you can create any Executor you want inheriting from the `Executor` class.
-In this case, our two executors are only printing a line.
+We will have more complex `Executors` later, for now our two `Executors` are only printing a line.
```

Comment:
We will have more complex `Executors` later. For now our two `Executors` are only printing a line.

(punctuation)